src/egon/data/datasets/gas_aggregation.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""
+The central module aggregating ch4 store caverns (stores) and ch4 generators 
+"""
+import pandas as pd
+import requests
+
+from egon.data import db
+from egon.data.datasets import Dataset
+
+
+class GasAggregation(Dataset):
+    def __init__(self, dependencies):
+        super().__init__(
+            name="GasAggregation",
+            version="0.0.0",
+            dependencies=dependencies,
+            tasks=(aggregate_gas),
+        )
+
+
+def aggregate_gas(scn_name="eGon2035"):
+    """Aggregation of ch4 stores and ch4 generators with same properties at the same bus.
+
+    Parameters
+    ----------
+    scn_name : str
+        Name of the scenario
+
+    Returns
+    -------
+
+    """
+    # Connect to local database
+    engine = db.engine()
+
+    components = [
+        # Generator
+        {
+            "columns": "scn_name, generator_id, bus, p_nom, carrier",
+            "table": "egon_etrago_generator",

```

Comment:
You should import "egon.data.datasets" to make reference to tables in the egon-data db. In this way, if it is necessary to change the name of a table, we just need to change that file.

Reply:
Fixed with https://github.com/openego/eGon-data/pull/632/commits/f06ff0ca55a0140a9bd6f8737f481697cd5052b2.